ktor-server/ktor-server-core/common/src/io/ktor/server/engine/CommandLine.kt

```diff
@@ -104,7 +104,7 @@ internal fun buildApplicationConfig(args: List<Pair<String, String>>): Applicati
     val configPaths = args.filter { it.first == "-config" }.map { it.second }
 
     val commandLineConfig = MapApplicationConfig(commandLineProperties)
-    val environmentConfig = getConfigFromEnvironment()
+    val environmentConfig = MapApplicationConfig(getKtorEnvironmentProperties())
```

Comment:
We could have an mpp library to work with Env variables :)

Reply:
Yeah, I was thinking about extracting it into `ktor-environment`.
Similar libraries:
* https://github.com/ScottPierce/kotlin-env-var
* https://github.com/luca992/getenv-kt
Though, they support less, than already available here.
Looks good as an issue in YT may be